back/scripts/utils/datagouv_api.py

```diff
@@ -170,22 +169,6 @@ def _next_page(url: str, params: dict | None = None) -> Tuple[dict, str]:
         return data.get("data", data), data.get("next_page")
 
 
-def normalize_formats_description(formats: pd.Series) -> str:
-    """
-    Classify with regex the various description of formats available on data.gouv into a set of fixed categories.
-    See `FORMATS_PATTERNS`for the list of patterns.
-
-    For example : "file:///srv/udata/ftype/csv" will be transformed into "csv".
-    """
-    matching = {
-        source: target
-        for pat, target in FORMATS_PATTERNS.items()
-        for source in formats.dropna().unique()
-        if re.search(pat, source.lower())
-    }
-    return formats.map(matching).fillna(formats)
-
```

Comment:
Du coup `FORMATS_PATTERNS` n'est plus utilisé sauf pour générer `IMPLEMENTED_FORMATS` ? Peut-être l'occasion de simplifier sa création.

Reply:
Replaced it with a class method of BaseLoader to determine parsable formats.